src/tests/readytorun/r2rdump/FrameworkTests/R2RDumpTester.cs

```diff
@@ -46,6 +46,8 @@ public void DumpCoreLib()
             string CoreLibAbsolutePath = Path.Combine(CoreRootVar, CoreLibFile);
             string OutputFile = Path.GetTempFileName();
             string TestDotNetCmdVar = Environment.GetEnvironmentVariable("__TestDotNetCmd");
+            // Unset COMPlus_GCName
```

Comment:
I assume the issue is that clrgc.dll is not available in the folder where for R2RDump runs, I think this is a fine fix. It would be nice if we can reword the comment to tell us why we did that in the code.

Reply:
correct, for now the workaround is to disable standalone gc while compiling the test. Will update comment.